packages/react-instantsearch/src/components/Breadcrumb.js

```diff
@@ -0,0 +1,92 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import Link from './Link';
+import classNames from './classNames.js';
+import translatable from '../core/translatable';
+
+const cx = classNames('Breadcrumb');
+
+const itemsPropType = PropTypes.arrayOf(
+  PropTypes.shape({
+    label: PropTypes.string.isRequired,
+    value: PropTypes.string.isRequired,
+  })
+);
+
+class Breadcrumb extends Component {
+  static propTypes = {
+    canRefine: PropTypes.bool.isRequired,
+    createURL: PropTypes.func.isRequired,
+    items: itemsPropType,
+    refine: PropTypes.func.isRequired,
+    rootURL: PropTypes.string,
+    separator: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
```

Comment:
That's what I mean, but for consistency, let's allow both until we don't have react 15 users anymore.